### PR TITLE
aalhommada-node.js-week2

### PR DESCRIPTION
**Pull request by aalhommada**

### CODE REVIEW

File: week2/homework/src/add.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+
+function add(toDoList) {
+  if (toDoList === undefined) {
+    console.log('please enter a valid value after the command add');
+  }
+ else {
+    fs.appendFile('./todoList.txt', toDoList + '\n', error => {
+      console.error(error);
```
Comment (gajduk) [inline week2/homework/src/add.js]: Do we want to log even when there is no error i.e. the error is null?
image

File: week2/homework/src/add.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+
+function add(toDoList) {
+  if (toDoList === undefined) {
+    console.log('please enter a valid value after the command add');
```
Comment (gajduk) [inline week2/homework/src/add.js]: proper handling of missing value 👍

File: week2/homework/src/help.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable indent */
+const fs = require('fs');
+
+function help(toDoList) {
+  fs.readFile('./help.txt', 'utf8', (err, data) => {
+    if (err) {
```
Comment (gajduk) [inline week2/homework/src/help.js]: proper error handling 👍

File: week2/homework/src/remove.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+
+function remove(index) {
+  fs.readFile('./todoList.txt', 'utf8', (err, data) => {
+    if (err) throw err;
```
Comment (gajduk) [inline week2/homework/src/remove.js]: It is not wrong to rethrow an error if you think it should be handled by the invoking function, but then you should be consistent. In other files you simply log the error and do not rethrow.

File: week2/homework/src/remove.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    const removeSelectedItem = data.split('\n');
+    removeSelectedItem.splice(index - 1, 1);
```
Comment (gajduk) [inline week2/homework/src/remove.js]: nice 👍

File: week2/homework/help.txt [hunk 6]
```diff
@@ -0,0 +1,7 @@
+
+help : Shows help section
```
[the remark this answers is not in the document]
Comment (aalhommada) [reply week2/homework/help.txt]: thanks

File: week2/homework/src/add.js (hunk 1 above)
Comment (aalhommada) [reply week2/homework/src/add.js]: I modify it

File: week2/homework/src/add.js (hunk 2 above)
Comment (aalhommada) [reply week2/homework/src/add.js]: thanks

File: week2/homework/src/remove.js (hunk 5 above)
Comment (aalhommada) [reply week2/homework/src/remove.js]: :)

File: week2/homework/src/remove.js (hunk 4 above)
Comment (aalhommada) [reply week2/homework/src/remove.js]: i change it , thanks for the info

**Pull request closed without merging**